src/main/java/org/example/CheckDivisionByThree.java

```diff
@@ -0,0 +1,28 @@
+package org.example;
+
+public class CheckDivisionByThree {
+
+    public static void validate(int[] values){
+        if(checkParameterForZero(values)){
+            for (int i = 0; i < values.length; i++){
+                try {
+                    if(values[i] % 3 != 0){
+                        System.out.println("Wrong value \"" + values[i] + "\"  not divisible by 3");
+                        throw new IllegalArgumentException();
+                    }else {
+                        System.out.println("Value \"" + values[i] + "\"  divisible by 3");
+                    }
+                }catch (IllegalArgumentException e){
+                    e.getMessage();
```

Comment:
po co łapiesz wyjątek który wyrzuciłaś kilka linii wyżej? on ma byc wyrzucony na zewnątrz

Reply:
ale tutaj w zadaniu trzeba było zrobić tak : jeżeli wyjątek zostanie rzucony powinien on zostać obsłużony a proces powinien iść dalej

Reply:
i jeśli to usunę, program się rozpadnie

Reply:
Bo może if powinien być wydzielony do oddzielnej metody np. checkIfDivisible(int number) i wtedy ta metoda będzie wyrzucać wyjątek IllegalArgumentException, a na zewnątrz w tej pętli for będziesz to łapała